doc/manual/manual.tex

```diff
@@ -10872,6 +10872,43 @@ \subsubsection{Yamauchi \& Takei anelastic shear wave velocity-temperature conve
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/brittle_thrust_wedges/velocity_bc_part.prm}
+
+A good and accurate solver convergence is always challenging to achieve in numerical thrust wedge models with a high spatial resolution (ca. 1 mm node spacing) and a large viscosity contrast. Here, we suggest that several parameters should be considered carefully. Firstly, the nonlinear and linear solver tolerances should be sufficiently strict to avoid numerical instabilities. Secondly, we use the discontinuous Galerkin method (set Use discontinuous composition discretization = true) to ensure that the discontinuous composition bound preserving limiter produces sharp interfaces between compositional layers. Lastly, using the harmonic averaging scheme for material and viscosity is required to achieve reasonable convergence behavior.
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/brittle_thrust_wedges/convergence_part.prm}
+
+\textbf{Experiment 1} tests whether model wedges in the stable domain of critical taper theory remain stable when translated horizontally. A quartz sand wedge with a horizontal base and a surface slope of 20 degrees is pushed 4 cm horizontally by inward movement of a mobile wall at the right boundary with a velocity of 2.5 cm/hour (Figure ~\ref{fig:btwexp1}).  The basal angle is zero (horizontal), a thin layer separates the sand and boundary to ensure minimum coupling between the wedge and bounding box during translation, and a sticky air layer is used above the wedge. Further, the purely plastic material should not undergo any deformation during translation.
+
+\begin{figure}
+\begin{center}
+  \centering
+  \includegraphics[width=\textwidth]{cookbooks/benchmarks/brittle thrust wedges/exp1.png}
+  \caption{\it Numerical model of a stable sand wedge. a) Initial model setup. b) Material field after 4 cm of translation. c) Strain rate field and d) pressure field.}
+  \label{fig:btwexp1}
+\end{center}
+\end{figure}
+
+\textbf{Experiment 2} tests how an unstable subcritical wedge deforms to reach the critical taper solution. In this experiment, horizontal layers of sand suffer 10 cm shortening by inward movement of a mobile wall with a velocity of 2.5 cm/hour (Figure ~\ref{fig:btwexp2}). It builds thrust wedges near the mobile wall through a combination of mainly in-sequence forward and backward thrusting.  The strain field highlights several incipient shear zones that do not always accumulate enough offset to become visible in the material field. The pressure field of the model remains more or less lithostatic, with lower pressure values in (incipient) shear zones.
```

Comment:
```suggestion
\textit{Experiment 2} tests how an unstable subcritical wedge deforms to reach the critical taper solution. In this experiment, horizontal layers of sand suffer 10 cm shortening by inward movement of a mobile wall with a velocity of 2.5 cm/hour (Figure~\ref{fig:btwexp2}). It builds thrust wedges near the mobile wall through a combination of mainly in-sequence forward and backward thrusting.  The strain field highlights several incipient shear zones that do not always accumulate enough offset to become visible in the material field. The pressure field of the model remains more or less lithostatic, with lower pressure values in (incipient) shear zones.
```

Reply:
"In this experiment, horizontal layers of sand suffer 10 cm shortening by inward movement of a mobile wall with a velocity of 2.5 cm/hour (Figure~\ref{fig:btwexp2}). It builds thrust wedges near the mobile wall through a combination of mainly in-sequence forward and backward thrusting." ->

"In this experiment, horizontal layers of sand **undergo** 10 cm shortening by inward movement of a mobile wall with a velocity of 2.5 cm/hour (Figure~\ref{fig:btwexp2}). **Model results show thrust wedge generation** near the mobile wall through a combination of mainly in-sequence forward and backward thrusting."